bundles/org.eclipse.e4.ui.css.swt/src/org/eclipse/e4/ui/css/swt/dom/ButtonElement.java

```diff
@@ -78,6 +78,9 @@ public boolean isPseudoInstanceOf(String s) {
 		if ("checked".equalsIgnoreCase(s)) {
 			return this.isSelected;
 		}
+		if ("selected".equalsIgnoreCase(s)) {
+			return this.isSelected;
+		}
```

Comment:
Merge this two

Reply:
Actually: is `checked` even valid? I couldn't find any usage in any CSS file

Reply:
I think it is valid, IIRC

Reply:
You're right, it's a valid selector: https://www.w3schools.com/cssref/sel_checked.php